recipes/data-management/cygnus/readme.md

```diff
@@ -7,14 +7,14 @@ Instructions on how to prepare your environment to test these recipes are given
 
 Moreover, we will be discussing the deployment of the agent as a single configurable entity. But note that within an agent, there exist multiple available configurations (using single and multiple sources, channels and sinks), as described in [Advanced Cygnus Architectures](http://fiware-cygnus.readthedocs.io/en/latest/architecture/index.html#advanced-cygnus-architectures). How you setup those internal advanced architectures and the advantages of each will not be covered here, since this is already discussed in the official documentation.
 
 That being said, in order to deploy Cygnus, we need to understand whether it's a stateless or stateful service. It turns out that source and sink parts of the agent do not persist data, however, the channels do, at least for a short period of time until data is processed and taken out of the channel by the sink[s]. As you can read from Cygnus and Flume's documentation, channels come in the form of MemoryChannel and FileChannel.
 
 It's easy to see that MemoryChannel and Batching has a potential for data loss, for example, in the case when an agent crashes before events were taken out of the channel. In fact, to avoid storage access for each event, Cygnus comes with default values of batch size and batch flush timeouts. As a side note, it'd be nice if this could be dynamically changed according to dynamic demands, but this is an interesting point to investigate at a later point.
 
-Therefore, the FileChannel could be used, at the cost of a higher latency, to give persistence to the "inflight data" and this way prevent a complete loss if there happens to be a software failure/crash at the agent. Note however that the location of this file within the container and not customizable, hence a container crash will cause those un-flushed values to be lost.
+Therefore, the FileChannel could be used, at the cost of a higher latency, to give persistence to the "inflight data" and this way prevent a complete loss if there happens to be a software failure/crash at the agent. Note however that the location of this file within the container and not customisable, hence a container crash will cause those un-flushed values to be lost.
```

Comment:
i wonder if it would not work to store the file channel on an nfs shared volume or something similar

Reply:
I think that may be an overkill considering the file is actually a process cache, but it may be worth the experiment. Feel free to add an issue if you'd like to consider it in roadmap

Reply:
(y)